lms/djangoapps/projects/views.py

```diff
@@ -295,17 +304,14 @@ def grades(self, request, pk):
 
         users = User.objects.filter(workgroups=pk)
         for user in users:
-            key = KeyValueStore.Key(
-                scope=Scope.user_state,
-                user_id=user.id,
-                block_scope_id=content_key,
-                field_name='grade'
+            module, created = StudentModule.objects.get_or_create(
```

Comment:
Nit: prefix the name `created` with an underscore if you're not going to use its value.